src/test/java/com/amazon/titan/graphdb/dynamodb/SingleDynamoDBGraphTest.java

```diff
@@ -66,7 +66,7 @@ public SingleDynamoDBGraphTest()
     @Override
     public WriteConfiguration getConfiguration() {
         final WriteConfiguration wc = super.getConfiguration();
-        final String methodName = name.getMethodName();
```

Comment:
The TestName rule is now in the AbstractDynamoDBGraphTest class, so no need to have the same rule overriding the base class in the implementing class.